apps/web/components/eventtype/AssignmentWarningDialog.tsx

```diff
@@ -39,7 +39,7 @@ const AssignmentWarningDialog = (props: AssignmentWarningDialogProps) => {
               router.replace(`/event-types/${id}?tabName=team`);
             }}
             color="minimal">
-            {t("go_back_and_assign")}
+            {t("go_back_assign_and_save")}
```

Comment:
I think we don't need to change it. In our app, we need to save maximum places in order to update, so it was understandable.

Reply:
Oh, ok!